plugins/SitesManager/angularjs/sites-manager/sites-manager-admin-sites-model.js

```diff
@@ -63,6 +75,16 @@
             fetchLimitedSitesWithAdminAccess();
         }
 
+        function lastPage()
+        {
+            sitesManagerAPI.getSitesIdWithAdminAccess(function (siteIds) {
```

Comment:
I haven't tested it, but I reckon it might not work when the user is currently searching for a site and then goes to the last page because I think here `getSitesIdWithAdminAccess ` would return really all sites whereas the number of results be lower when searching.

Reply:
Hi, yes you are right.